app/views/chargeback/_cb_assignments.html.haml

```diff
@@ -65,3 +50,50 @@
                 :javascript
                   miqInitSelectPicker();
                   miqSelectPickerEvent("#{@edit[:new][:cbshow_typ]}__#{id}", "#{url}")
+              - data[:subtenant].each do |tenant|
+                %tr{:'id' => 'tenant_' + tenant[:id].to_s , :'data-parent' => '#tenant_' + tenant[:parent].to_s}
+                  %td.treegrid-node
+                    = h(tenant[:name])
+                  %td
+                    - options = @edit[:cb_rates].invert.sort
+                    = select_tag("#{@edit[:new][:cbshow_typ]}__#{tenant[:id]}",
+                                options_for_select([[nothing, "nil"]] + options, @edit[:new]["#{@edit[:new][:cbshow_typ]}__#{tenant[:id]}".to_sym].to_s),
+                                "data-miq_sparkle_on" => true, "data-miq_sparkle_off" => true, :class    => "selectpicker")
+                  :javascript
+                    miqInitSelectPicker();
+                    miqSelectPickerEvent("#{@edit[:new][:cbshow_typ]}__#{tenant[:id]}", "#{url}")
```

Comment:
You can omit the interpolation and the quotation marks around `url`, they're working against each other!